community/kernel/src/main/java/org/neo4j/kernel/impl/store/CommonAbstractStore.java

```diff
@@ -851,18 +851,6 @@ protected void assertNotClosed()
     @Override
     public void close()
     {
-        if ( idGenerator == null || !storeOk )
-        {
-            try
-            {
-                closeStoreFile();
```

Comment:
you can freely do it in an active brunch, no reason to touch something else there in general just because you can, that's why it's called maintenance